common/messages/messages_en.go

```diff
@@ -188,7 +188,7 @@ var messagesEn = map[string]util.Message{
 		Code:        VolumeAccessPointExist,
 		Description: "The volume ID '%s' could not be deleted from your VPC. Volume has access points which needs to deleted.Please go through the list of VPCs = '%v'",
 		Type:        util.DeletionFailed,
-		Action:      "User need to review all access points and delete them first before deleting volume.", //TODO once cli will be ready then we need to update this error message
+		Action:      "User need to review all access points and delete them first before deleting volume. Run `ibmcloud is share-targets <VOLUME_ID>` to get the volume access point for the file share. ",
```

Comment:
Run ibmcloud is share-targets <VOLUME_ID> to list all volume access point for a volume.